server/ai_live_video.go

```diff
@@ -355,6 +363,29 @@ func startTrickleSubscribe(ctx context.Context, url *url.URL, params aiRequestPa
 			clog.V(8).Info(ctx, "trickle subscribe read data completed", "seq", seq, "bytes", humanize.Bytes(uint64(n)), "took", time.Since(copyStartTime))
 		}
 	}()
+
+	// watchdog: fires if orch does not produce segments for too long
+	go func() {
+		for {
+			select {
+			case <-segmentTicker.C:
+				// check whether this timeout is due to missing input
+				params.liveParams.mu.Lock()
+				lastInputSegmentTime := params.liveParams.lastSegmentTime
```

Comment:
What's the initial value of `lastSegmentTime`? If it's `0` by default, then won't it cause a problem that is suspend the Orchestrator just at the beginning of the processing?